### PR TITLE
Pull request for #90: Load PYTHONPATH in SCons ENV

### PR DESCRIPTION
**Pull request by arosenbe**
@stanfordquan, can you review this quick fix from #90? Thanks!

### CODE REVIEW

**Comment by stanfordquan** [comment]
@arosenbe , can I see the documentation for this exact bug? Where is this `os.environ('PYTHONPATH')` variable from? If I run this under native/virtualenv Python as opposed to Mac native Python what do I get? I'd also like to test this in Windows a bit. 

My intuition is that this is not as naive a fix as we're making it out to be and has potential to become another failure point, so I'd like to be careful about this.

File: paper_slides/SConstruct [hunk 1]
```diff
@@ -8,14 +8,18 @@ sys.dont_write_bytecode = True # Don't write .pyc files
...
 
 # Start log
 gs.log.start_log(mode, vers)
 
 # Define the SCons environment
-env = Environment(ENV = {'PATH': os.environ['PATH']}, 
+try:
+   pythonpath = os.environ['PYTHONPATH']
+except:
+   pythonpath = ''
```
Comment (stanfordquan) [inline paper_slides/SConstruct]: We're defining `pythonpath` twice in this file.

Comment (arosenbe) [reply paper_slides/SConstruct]: Whoops, forgot to delete this block.

**Comment by arosenbe** [comment]
@stanfordquan, here's some information. Let me know if you still want more information. 

### The exact bug

I brewed my python, which redirected install location for pip. I set PYTHONPATH on my system so that python checks this new install location, rather than the default. 

screen shot 2017-11-10 at 1 12 44 pm

When I run SCons, the PYTHONPATH environment variable isn't set, so python doesn't know to look in this new install location. This creates bugs when I try to import modules that have been `pip install`ed: PyYaml, pandas, etc. You can see the difference in this snapshot from my builder log

screen shot 2017-11-10 at 1 14 27 pm

SCons tries to execute python scripts under the environment defined by the env variable. Since we don't currently load PYTHONPATH into the env variable, I can't execute python with SCons in the way that I normally do on my system. I fix this by loading PYTHONPATH into env. 

### Where PYTHONPATH is from
[PYTHONPATH](https://docs.python.org/2/using/cmdline.html#envvar-PYTHONPATH) is an environment variable that prepends directories to python's search hierarchy when searching for packages. In the same way that you'd add your python executable to PATH, you can add directories with python packages to the PYTHONPATH.

### Worries about failures

I'm happy to test under a variety of settings, but I don't expect any failures. PYTHONPATH doesn't exist by default, and in this case we ignore it. If it does exists, then we load it.

**Comment by stanfordquan** [comment]
Alright, go for it!

**Pull request merged**